app.php

```diff
@@ -407,6 +407,7 @@ public function load_admin_scripts($hook) {
 					'backbone',
 					'backbone.trackit',
 					'_ca_condition-groups',
+					'vaa-view-admin-as',
```

Comment:
Maybe create this compatibility function the other way around?
Instead of removing all scripts by default, only remove the scripts you know are not compatible?